mikeio1d/result_network/result_location.py

```diff
@@ -30,12 +30,31 @@ class ResultLocation:
     def __init__(self, data_items, res1d):
         self.data_items = data_items
         self.res1d = res1d
-        self.quantity_label = 'q_'
+        self.quantity_label = "q_"
         self.result_quantity_map = {}
         self.element_indices = None
+        self._static_attributes = []
+
+    def __repr__(self) -> str:
+        return f"<{self.__class__.__name__}>"
+
+    def _repr_html_(self) -> str:
+        repr = build_html_repr_from_sections(
+            self.__repr__(),
+            [
+                ("Attributes", {k: getattr(self, k) for k in self._static_attributes}),
+                ("Quantities", list(self.result_quantity_map.keys())),
+            ],
+        )
+        return repr
+
+    def set_static_attribute(self, key, value):
```

Comment:
I thought in order to wrap the properties of .NET objects we could use properties in Python. Then the static items would be even more "static". However, I can see that _static_attributes list is also used to create a representation, so this may be better.